docs/adguard-vpn-for-ios/solving-problems/access-issues.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Usage issues with subscriptions purchased via the App Store 
+sidebar_position: 4
+---
+
+Some users can't use their AdGuard VPN subscription purchased through the App Store. This issue may occur in two cases: 
+
+*  if you use a different account for the Apple Store purchases 
+*  or if you have hidden your email address from the app developers 
+
+To resolve this issue and gain access to your account, please follow these steps:
+
+1.  Go to Settings → Apple ID → iTunes & App Store → View Apple ID
+2.  Verify that the email address matches the one you use for your Apple ID
+3.  If the email addresses are different, please provide us with the email address used for the App Store. Send it to `support@adguard.com`
+4.  If the email addresses match, navigate to Apple ID → iCloud → Hide My Email, find our app in the list, copy the email address, and send it to `support@adguard.com`. It typically ends with "@privaterelay.appleid.com"
+
+Once the support team has the email address you used for the purchase, they'll transfer your subscription to the correct account and you'll be able to start using it. Or they will ask to send some additional information. You can provide the support team with your purchase receipt from Apple right away to speed up the processing of your request. 
```

Comment:
Or they will ask to send some additional information. You can provide the support team with your purchase receipt from Apple right away to speed up the processing of your request.
—>
Or they may ask for additional information. You can provide the support team with your Apple purchase receipt to help speed up the process.